sentry-opentelemetry/sentry-opentelemetry-core/src/main/java/io/sentry/opentelemetry/SentrySpanExporter.java

```diff
@@ -134,51 +129,9 @@ private boolean isSpanTooOld(final @NotNull SpanData span, final @NotNull Sentry
   }
 
   private @NotNull List<SpanData> filterOutSentrySpans(final @NotNull Collection<SpanData> spans) {
```

Comment:
Are there instances where `SentrySampler` is not invoked and spans need to be filtered here in the exporter?

Reply:
Theoretically it's possible the span is modified during runtime and then has different attributes in `SpanExporter`.